packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/sagas.js

```diff
@@ -54,7 +54,8 @@ export default ({ api, coreSagas }) => {
     coreSagas
   })
 
-  const registerUserCampaign = function*(newUser = false) {
+  const registerUserCampaign = function*(payload) {
```

Comment:
not exactly the same formatting that we use everywhere, it's usually `payload` property having everything inside